plugin-maven/README.md

```diff
@@ -523,7 +523,7 @@ You can use prettier in any language-specific format, but usually you will be cr
           </property>
         </devDependencyProperties>
         <!-- Specify config file and/or inline config, the inline always trumps file -->
-        <configFile>${basedir}/path/to/configfile</configFile>
+        <configFFile>${basedir}/path/to/configfile</configFile>
```

Comment:
Was this changed on purpose?

Reply:
:facepalm: sorry for that

I'll remove this again.

Reply:
@thc202 change is now removed again